snap/local/snapcraft.yaml.template

```diff
@@ -154,6 +154,19 @@ parts:
     stage:
       - usr/lib/nordvpn/openvpn
 
+  libraries:
+    plugin: dump
+    source-type: local
+    source:
+      - on amd64: ./bin/deps/lib/amd64/
```

Comment:
This template file is supposed to be used as a workaround for building `snap` package for different architectures - source path to corresponding architecture binary is replaced from variable during preprocessing. See line #142

Reply:
Great catch. I addressed it here https://github.com/NordSecurity/nordvpn-linux/pull/575/files#diff-605cd881227522280b3e29a4cda48bf254060d476155901d307c9d5386841303R161